runtime/runtime/tests/test_async_calls.rs

```diff
@@ -34,7 +35,7 @@ fn test_simple_func_call() {
         vec![Action::FunctionCall(Box::new(FunctionCallAction {
             method_name: "sum_n".to_string(),
             args: 10u64.to_le_bytes().to_vec(),
-            gas: GAS_1,
+            gas: Gas::from_gas(GAS_1),
```

Comment:
Instead of converting GAS_1 into Gas type here, let's make the const already of Gas type.